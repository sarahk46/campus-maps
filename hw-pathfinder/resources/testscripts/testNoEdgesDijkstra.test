# Find a path with Dijkstra's algorithm when
# no edges are present in the graph.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

FindPath simpleGraph A B