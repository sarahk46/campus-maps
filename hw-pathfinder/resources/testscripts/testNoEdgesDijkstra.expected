# Find a path with Dijkstra's algorithm when
# no edges are present in the graph.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

path from A to B:
no path found